src/Employer/Employer.Web/Orchestrators/EditVacancyDatesOrchestrator.cs

```diff
@@ -40,15 +39,18 @@ public async Task<Vacancy> GetVacancyAsync(VacancyRouteModel vrm)
 
             Utility.CheckAuthorisedAccess(vacancy, vrm.EmployerAccountId);
 
+            if (vacancy.CanExtendStartAndClosingDates == false)
+                throw new InvalidStateException(string.Format(ViewModels.ErrorMessages.VacancyNotAvailableForEditing, vacancy.Title));
```

Comment:
Would suggest a different error message such as `The dates of vacancy '{what's my name}' can't be edited.` as the current NotAvailableForEditing is for something slightly different of when the vacancy has been edited by another person. Maybe that one itself needs renaming.

Reply:
👍